src/web/components/Dropdown.tsx

```diff
@@ -261,29 +271,29 @@ export const Dropdown = ({ id, label, links, dataLinkName }: Props) => {
 					>
 						{label}
 					</button>
-					<ul
-						className={cx({
-							[ulStyles]: true,
-							[ulExpanded]: isExpanded,
-						})}
-						data-cy="dropdown-options"
-					>
-						{links.map((l, index) => (
-							<li key={l.title}>
-								<a
-									href={l.url}
-									className={cx({
-										[linkStyles]: true,
-										[linkActive]: !!l.isActive,
-										[linkFirst]: index === 0,
-									})}
-									data-link-name={l.dataLinkName}
-								>
-									{l.title}
-								</a>
-							</li>
-						))}
-					</ul>
+					<div className={isExpanded ? displayBlock : displayNone}>
+						{children ? (
+							<>{children}</>
+						) : (
+							<ul className={ulStyles} data-cy="dropdown-options">
+								{links.map((l, index) => (
+									<li key={l.title}>
+										<a
+											href={l.url}
+											className={cx({
+												[linkStyles]: true,
+												[linkActive]: !!l.isActive,
+												[linkFirst]: index === 0,
+											})}
+											data-link-name={l.dataLinkName}
+										>
+											{l.title}
+										</a>
+									</li>
+								))}
+							</ul>
+						)}
+					</div>
```

Comment:
This component could benefit from some refactoring, we are getting quite nested turnaries that are become a little hard to follow

Reply:
Yeah, agreed. This marks the first time we're using this component in another place and I noticed things could be arranged better for this, the styles in `Dropdown` are too specific and could be moved up. I think as well there needs to be some investigation around the noJs route.

I'll ticket this up for another PR.

Reply:
https://trello.com/c/YHajezB2/2763-refacctor-dropdown